test/ETHBankContract.js

```diff
@@ -12,15 +14,121 @@ describe("ETHBank Test Suite", function () {
 
   describe("Post Deployment State Variables", async () => {
     it("Should return state variables", async () => {
-        // extract loadFixture variables
+      // extract loadFixture variables
       const { ETHBankContract, owner, addr1, addr2 } = await loadFixture(deployTokenFixture);
-      expect(await ETHBankContract.owner()).to.eq(owner.address)
-      expect(await ETHBankContract.ethBalances(owner.address)).to.eq(0)
+      expect(await ETHBankContract.owner()).to.eq(owner.address);
+      expect(await ETHBankContract.ethBalances(owner.address)).to.eq(0);
     });
   });
 
-  describe("Transactions", async () => {
+  describe("Deposit Validations", async () => {
```

Comment:
Under `Transactions`, you should have sub-wrappers - `Deposit` and `Withdrawal`
To appropriately categorize your test cases, under each of these, ensure you have corresponding `Validation`, `Transactions` and `Events` `describe` wrappers and each with its `it` test case